docs/sources/enterprise/access-control/roles.md

```diff
@@ -44,6 +44,11 @@ Role names must be unique within an organization.
 
 Roles with names prefixed by `fixed:` are fixed roles created by Grafana and cannot be created or modified by users.
 
+### Role Display name
```

Comment:
```suggestion
### Role display name
```
Use sentence case. For more information, see https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings.